Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/PokePointer.cs

```diff
@@ -13,7 +13,28 @@ public class PokePointer : BaseControllerPointer, IMixedRealityNearPointer
 
         [SerializeField]
         protected GameObject visuals;
-        
+
+        /// <summary>
+        /// Maximum distance a which a touchable surface can be interacted with.
+        /// </summary>
+        [SerializeField]
+        protected float touchableDistance = 0.2f;
```

Comment:
I'm not clear what the sentence means either ... The touch system is convoluted and inconsistent TBH. I'll try to explain what happens:

_Reachable Objects_ are objects with a both a `BaseNearInteractionTouchable` and a collider within `TouchableDistance` from the poke pointer (based on [OverlapSphere](https://docs.unity3d.com/ScriptReference/Physics.OverlapSphere.html)).

If a poke pointer has no _active_ touchable (`PokePointer.currentTouchableObjectDown`), then it will try to select one from the Reachable Objects based on:
1. Layer mask priority: Lower-priority layer masks will only be considered if higher-priority layers don't contain any Reachable Objects.
1. Distance: the closest object in the highest priority layers is selected based on `BaseNearInteractionTouchable.DistanceToTouchable`
1. Poke Threshold: The object becomes active once it crosses the `PokeThreshold` distance (behind the surface). At this point the `OnTouchStarted` or `PointerDown` event is raised.

If  a poke pointer _does_  have an _active_ touchable it will not consider any other object, until the `DistanceToTouchable` exceeds the `DebounceThreshold` (in front of the surface). At this point the active object is cleared and the `OnTouchCompleted` or `PointerUp` event is raised.

I really don't know how to squeeze this complexity into a single comment, and rewriting the touch system is out of scope of this PR imho.

Reply:
Thanks for that explanation. It was actually really helpful. Any reason we wouldn't want to have such a long comment in the code? I actually think it would be quite helpful for people trying to understand the touch input system.

Reply:
Fair enough, i'll add it somewhere prominent for people to find.

Reply:
I've added this as inline documentation on the PokePointer class now, with full xrefs to the mentioned properties and functions.